### PR TITLE
fix(keeper): require recent scheduled proof evidence

### PR DESCRIPTION
**Pull request by jeong-sik**
## Summary

- Make `scheduled_proactive_autonomy` respect `--window-hours` instead of crediting lifetime `proactive_count_total` forever.
- Require meta-only scheduled proof to be recent and non-error, because failed scheduled attempts update `last_proactive_ts` for cooldown but do not prove the feature works.
- Surface per-keeper `meta_last_proactive_ts`, `meta_last_proactive_outcome`, and whether meta evidence satisfies the active proof window.
- Add regressions proving stale meta-only evidence and recent error meta do not satisfy a 24h scheduled-proactive proof.
- Repair the current-main unmatched parenthesis in `lib/keeper/keeper_turn.ml` that blocked the focused Dune target after rebasing.

## Why this matters

The objective is to prove whether real keepers are autonomously using every feature. A strict 24h proof cannot rely on stale lifetime scheduled counters from days ago, and it also cannot treat failed scheduled attempts as success. With this branch, the report now reflects live scheduled-proactive recency and outcome quality instead of stale or failed counters.

Latest current-runtime branch proof artifact:

- `/private/tmp/keeper_feature_proof_20260506_current3_recency_branch.json`
- status: `warn`
- summary: 18 features, 9 pass / 9 warn / 0 fail / 9 gaps
- scheduled proactive: `2/17 proactive-enabled keepers have scheduled proactive evidence in the last 24.0h`
- observed scheduled keepers: `taskmaster`, `verifier`
- recent scheduled attempts excluded as proof: `jobsian_purist` and `tech_glutton` have `meta_last_proactive_outcome=error` / decision-log failures

Earlier same-branch snapshots were useful but less strict: the first showed `0/17`, then a rerun showed `3/17` before this patch rejected failed scheduled meta. The remaining truth is still not all-green: 15 proactive-enabled keepers are missing successful 24h scheduled-proactive proof, and the other weak/missing feature groups still need controlled live exercises or additional repair.

## Verification

- `git diff --check origin/main..HEAD`
- `DUNE_CACHE=disabled MASC_SKIP_PIN_CHECK=1 MASC_SKIP_OPAM_LOCK=1 MASC_SKIP_DEPS_CHECK=1 MASC_DUNE_THROTTLE=0 scripts/dune-local.sh build test/test_dashboard_keeper_feature_proof.exe bin/keeper_feature_proof_report.exe`
- `_build/default/test/test_dashboard_keeper_feature_proof.exe` (`10 tests run`)
- `_build/default/bin/keeper_feature_proof_report.exe --base-path /Users/dancer/me --window-hours 24 --strict > /private/tmp/keeper_feature_proof_20260506_current3_recency_branch.json` exits 2 because strict proof still has gaps.

## Strict Proof Status

This PR does not make all features pass. It makes the proof stricter and shows the remaining all-feature claim is not achieved: scheduled proactive evidence is partial in the 24h window, and the other weak/missing feature groups still require controlled live exercises or additional repair.

### CODE REVIEW

File: test/test_dashboard_keeper_feature_proof.ml [hunk 1]
```diff
@@ -191,6 +199,17 @@ let write_scheduled_decision config keeper_name =
       ];
```
[the remark this answers is not in the document]
Comment (jeong-sik) [reply test/test_dashboard_keeper_feature_proof.ml]: Addressed in d315180445: the helper now writes a deterministic ts_unix/ISO pair from the test epoch, and test_decision_log_counts_as_scheduled_proof passes a fixed now after that timestamp.

**Review by anyang-keepers — COMMENTED** [review]
MASC improver check: three observations on this PR.

**1. keeper_turn.ml parenthesis fix (line-level):** The 1-line repair to `lib/keeper/keeper_turn.ml` is correct and low-risk — it unblocks the focused Dune target after a rebase conflict. Good catch including it in this branch rather than letting it block independently.

**2. scheduled_proactive_autonomy windowing logic (structural):** Replacing lifetime `proactive_count_total` with a `--window-hours`-bounded check is the right semantic shift. The proof system's purpose is to verify *live* capability, not historical activity. The exclusion of `meta_last_proactive_outcome=error` from proof is critical — without it, failed scheduled attempts (which update `last_proactive_ts` for cooldown) would falsely satisfy the proof. The 24h window produces the honest result: 2/17 keepers have valid proof. This is a feature, not a bug.

**3. Test coverage (verification):** The two regression tests (stale meta-only evidence + recent error meta exclusion) directly cover the two failure modes that motivated this change. 10 tests passing is consistent. One suggestion: consider adding a third regression for the boundary case where `meta_last_proactive_ts` is exactly at the window edge (e.g., 23h59m ago) to ensure the comparison uses strict `>` rather than `>=` for the recency check — this prevents off-by-one proof satisfaction at the window boundary.
